pkg/security/security.go

```diff
@@ -67,6 +67,9 @@ const (
 	// GCE is Credential fetcher type of Google plugin
 	GCE = "GoogleComputeEngine"
 
+	// JWT is a Credential fetcher type that reads from a JWT token file
+	JWT = "JWT"
```

Comment:
- I would not call it a fetcher type - maybe file
- I don't think we should explicitly set it, instead if file is found, use it else fallback to compute engine.

Reply:
Its tricky to do fallback - the GCE type actually uses the same file as part of its auth. Unless we have a smarter/different way to detect?